marine_sensor_msgs/bmr/RadarSector.bmr

```diff
@@ -0,0 +1,421 @@
+class update_marine_sensor_msgs_RadarScanline_8621d97f7489a206598e189d17efed48(MessageUpdateRule):
```

Comment:
Will the bag migration be grumpy about the formatting / comment changes you made in response to one of my earlier comments? 

If we need to update these, we should hold off until the rest of the MR is stabilized, then do the bmr in a final pass.

Reply:
A rosbag check on an old bag file suggests that all the required migration rules can be found, so it seems like we'll be all set. I'll do a final check and actually convert a file when we think we have something ready to merge.

Reply:
Yay! Glad it's robust in that way.